### DISCUSSION TITLE
持槍真的有合法這回事嗎??

### DISCUSSION THREAD

**Discussion by WeiYen1103** [1101-GS4719-C/Python-Community-GS4719-C#1696] [General]
暑假時許多人都有選擇前往泰國旅遊，但其實泰國已實行持槍合法化了，前陣子有一篇新聞，震驚了我和我的室友，最近又開始跟室友們聊天聊到這件事，泰國14歲少年持槍進到曼谷商場並且無差別攻擊，甚至造成了死亡，這件事又再次點出泰國槍枝氾濫的犯罪問題。最近，也就是在昨天，10/31日本的琦玉縣也在醫院爆發槍擊案，甚是有人質被扣押。這些事情都讓我再次思考，國家的往年歷史，真的可以列為國家是否可以槍枝合法的依據嗎?
人性是真的本善嗎?
當人類開始掌握了權力，手中持有可以恫嚇他人生命及財產的武器時，真的能保持善良，真的能只把這份危險的武器當作自保用嗎?
我認為，不。
這種槍枝 炸藥會危及他人生命安全的武器，我認為就應該要受到保護管束，而非人人皆可持有，因為我認為這樣只會造成人心惶恐，造成更多社會案件。

**Comment by ATREE01** [comment]
我認為這樣的說法並不正確，因為並不是所有開放持有槍枝的國家都是非常危險的，隨然像是美國之類的地方很常聽到有槍擊事件，但是像是瑞士，也是人民能夠合法持槍的國家卻很少發生像美國那樣的大型槍擊案，所以我覺得主要還是會跟教育以及整體國民水準有關。

**Comment by Yanxxx7979** [comment]
認同樓上所說的，一樣東西會被如何使用，完全倚仗個人。版主所說「手中持有可以恫嚇他人生命及財產的武器時」是否能保持善良，這不只是到的應該規範也是法律理應規範約束的。因此我認為在健康的社會附帶健全的法律之下，持槍合法化並非不可。只是若是以台灣來說，以我的看法，要走到這，還需要不少時間。

**Comment by mrweixd** [comment]
持槍是否合法取決於國家的法規。只要當地法律允許，那就是有合法這回事，所以感覺你問的問題蠻怪的。如果你的問題是持槍是否該合法的話，那我認為上面幾樓的說法非常具有參考價值。
